build.gradle

```diff
@@ -1,11 +1,12 @@
-buildscript {
-    repositories {
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:0.6'
-    }
+plugins {
+    id 'groovy'
+    id 'idea'
+    id 'maven-publish'
```

Comment:
No need for this one I believe. The Bintray plugin already applies it internally.

Reply:
OK